parlai/core/distributed_utils.py

```diff
@@ -15,14 +15,11 @@
 import builtins
 import pickle
 import contextlib
+from parlai.core.utils import check_torch_version
 
-try:
-    import torch.version
-    import torch.distributed as dist
-
-    TORCH_AVAILABLE = True
-except ImportError:
-    TORCH_AVAILABLE = False
+check_torch_version()
+import torch  # noqa: E402
```

Comment:
you can remove the ones that are marked `noqa: E402`. They were there only for the import test.

This dist one shouldn't be E402 though.

Reply:
I had removed the previous torch statements having ***```# noqa: F401```***.
The ```noqa: E402``` was added by me in order to ignore the lint error that says module level import not at top of module. Also, we need torch in this module in areas like ```torch.cuda.ByteTensor(buffer_size)```